src/scheduler/pbs_sched.c

```diff
@@ -732,7 +732,7 @@ lock_out(int fds, int op)
 	flock.l_start  = 0;
 	flock.l_len    = 0;	/* whole file */
 	if (fcntl(fds, F_SETLK, &flock) < 0) {
-		pbs_strncpy(log_buffer, "pbs_sched: another scheduler running\n", LOG_BUF_SIZE);
+		pbs_strncpy(log_buffer, "pbs_sched: another scheduler running\n", sizeof(log_buffer));
 		log_err(errno, msg_daemonname, log_buffer);
```

Comment:
This is a static string, mind skipping the strcpy() and moving it directly into the log_err()?

Reply:
I've checked in all places, there isn't an instance where I'm doing sizeof on a pointer pointing to dynamically allocated memory.